user-edit.css

```diff
@@ -34,4 +34,24 @@
 
 .two-factor-methods-table tbody tr:nth-child(odd) {
 	background-color: #f9f9f9;
-}
\ No newline at end of file
+}
+
+@media print {
+	body * {
+		visibility: hidden;
+		max-height: 100px;
```

Comment:
This was an attempt to make sure it didn't print 2-3 extra blank pages. It didn't work however. In my print preview it still shows 2 extra pages.